### ISSUE TITLE
Running README Example Fails

### ISSUE THREAD

**Issue by alallier** [mhsjlw/imdb#2]
When I try and run the readme example 

```
var imdb = require('imdb');

imdb('tt3659388', function(err, data) {
    if(err)
        throw err;

    console.log(data);
});
```

I get the following error
```
D:\Documents\GitHub\imdb\node_modules\imdb\lib\imdb.js:14
            var year = body.match(/<span class="nobr">\(<a href="\/year\/\d+\/\?ref_=tt_ov_inf">(\d+)<\/a>\)<\/span>/i)[1];
                                                                                                                       ^

TypeError: Cannot read property '1' of null
    at Request._callback (D:\Documents\GitHub\imdb\node_modules\imdb\lib\imdb.js:14:120)
    at Request.self.callback (D:\Documents\GitHub\imdb\node_modules\imdb\node_modules\request\request.js:199:22)
    at emitTwo (events.js:87:13)
    at Request.emit (events.js:172:7)
    at Request.<anonymous> (D:\Documents\GitHub\imdb\node_modules\imdb\node_modules\request\request.js:1036:10)
    at emitOne (events.js:82:20)
    at Request.emit (events.js:169:7)
    at IncomingMessage.<anonymous> (D:\Documents\GitHub\imdb\node_modules\imdb\node_modules\request\request.js:963:12)
    at emitNone (events.js:72:20)
    at IncomingMessage.emit (events.js:166:7)
 Program node app.js -dev exited with code 1
```

**Comment by mhsjlw** [comment]
Hi! This is due to imdb's website change making the old version incompatible. I plan on fixing this later today. I will close this issue once I update the module.

**tldr;**: It will be fixed later today.

**Comment by alallier** [comment]
Okay great thanks for the quick reply.

**Comment by mhsjlw** [comment]
Something interesting to point out:

Some movies on imdb take different formats, for example yours: http://www.imdb.com/title/tt3659388/ and http://www.imdb.com/title/tt0066995/ look and are different. I'm working to figure out the 2 different formats and extract appropriately though.

**Comment by mhsjlw** [comment]
Just a status update, I believe I have fixed it, I am just testing it out to make sure it works properly and that I won't run into any more problems

**Comment by mhsjlw** [comment]
Make sure you update the module to version `1.0.0` and if you run into any further problems (which you shouldn't!) just create a new issue!

**Comment by mhsjlw** [comment]
Oops! It seems that I've broken it again... when you use unreleased movies, my genre hack doesn't work. I'll fix it right away.

**Comment by mhsjlw** [comment]
It's fixed.

**Comment by alallier** [comment]
Thanks for your quick and diligent work on this!

**Comment by mhsjlw** [comment]
No problem. If you have any further issues, let me know.
